### ISSUE TITLE
Fix Request  for False Positive in AvoidMutableLists

### ISSUE THREAD

**Issue by Mugil** [jborgers/PMD-jPinpoint-rules#164]
When the Values for bean are loaded using @ConfigurationProperties the issue is reported falsely by PMD rules

```
@ConfigurationProperties(prefix = "joe")
public class LoadEmpConfigs {
    private final List<String> lstEmpConfigs = new ArrayList<String>();
	.
	.
	.
}
```

The lstEmpConfigs values are read from file joe.yml using a CSV Design Pattern to read file and set in list whereas PMD looks for a setter which is not available in code.
